src/Visitor/MutationsCollectorVisitor.php

```diff
@@ -59,24 +59,24 @@ final class MutationsCollectorVisitor extends NodeVisitorAbstract
 
     private $mutators;
     private $filePath;
-    private $fileAst;
+    private $fileNodes;
     private $codeCoverageData;
     private $onlyCovered;
 
     /**
      * @param Mutator[] $mutators
-     * @param  Node[] $fileAst
+     * @param Node[]    $fileNodes
```

Comment:
since when we started doing it? I would like to avoid it as adding one more parameter with a longer type will lead to indenting **all* the rows to the right

Reply:
I think it's my IDE, looks like it's not something enforced by the CS fixer yet